[Pages] docusaurus site guide (#6038)
cloudflare/cloudflare-docs: content/pages/framework-guides/deploy-a-docusaurus-site.md

In your terminal, `cd` into your new Docusaurus project directory and run:

```sh
# Set up the local repository
$ git init
$ git remote add origin <YOUR_NEW_GITHUB_REPOSITORY_URL>
$ git branch -M main

# Commit all initial files
$ git add -A
$ git commit -m "<COMMIT_MESSAGE>"

# Send commit to new GitHub repository
$ git push -u origin main
```

You have successfully created a GitHub repository and pushed your Docusaurus project to that repository.

## Deploy with Cloudflare Pages

Deploy your site to Pages:

1. Log in to the [Cloudflare dashboard](https://dash.cloudflare.com/).
2. In **Account Home**, select **Pages** > **Create a project**. 
3. Select the new GitHub repository that you created and, in the **Set up builds and deployments** section, select *Docusaurus* as your **Framework preset**. Your selection will provide the following information.

{{<table-wrap>}}

| Configuration option | Value                       |
| -------------------- | --------------------------- |
| Production branch    | `main`                      |
| Build command        | `npm run build`             |
| Build directory      | `build`                     |

{{</table-wrap>}}

> 4. Go to **Environment variables (advanced)** and create a `NODE_VERSION` variable with a value of `16`.

Author: ```suggestion
4. Select **Environment variables (advanced)** > **Add variable** > create a `NODE_VERSION` variable with a value of `16`.
```
